commands/translate.js

```diff
@@ -0,0 +1,25 @@
+const fs = require('fs');
+const readline = require('readline');
+const { appFolder } = require('../constants');
+const readStream = fs.createReadStream([appFolder, 'file.dat'].join("/"));
+
+exports.command = 'translate [key]';
```

Comment:
I believe the commands should be abstracted away in one of their own modules.
Here is a link to think about that, but I think you all ready know I believe:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules